dvc/fs/dvc.py

```diff
@@ -75,28 +75,25 @@ class _DvcFileSystem(AbstractFileSystem):  # pylint:disable=abstract-method
 
     root_marker = "/"
 
-    PARAM_REPO_URL = "repo_url"
-    PARAM_REPO_ROOT = "repo_root"
-    PARAM_REV = "rev"
-    PARAM_CACHE_DIR = "cache_dir"
-    PARAM_CACHE_TYPES = "cache_types"
-    PARAM_SUBREPOS = "subrepos"
-
     def __init__(
```

Comment:
Changing the signature of the function, to make it easier to pass `url`. Surprisingly, no tests failed.